### PR TITLE
HUDs now craftable

### PR DESCRIPTION
**Pull request by Klaypexx**
Sec, medical, chemical huds now craftable.
Also sec hud added to sec vending machine

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Sanchez_
- add: NT finally got a license to create HUDs in techfab

### CODE REVIEW

**Comment by EmoGarbage404** [comment]
- chemical analysis already is printable. It's just in the protolathe.
- medsec needs to at the very least be research

**Comment by Holinka4ever** [comment]
Wait, isnt chemistry hud is exsclusive for researching?

**Comment by Klaypexx** [comment]
> * chemical analysis already is printable. It's just in the protolathe.
> * medsec needs to at the very least be research

yep, maybe chemical hud move it to medfab?

File: Resources/Prototypes/Recipes/Lathes/security.yml [hunk 1]
```diff
@@ -422,3 +422,21 @@
...
     Plastic: 950
+
+- type: latheRecipe
+  id: ClothingEyesHudSecurity
+  result: ClothingEyesHudSecurity
+  name: security hud
+  completetime: 2
+  materials:
+    Glass: 200
+    Steel: 300
```
Comment (ficcialfaint) [inline Resources/Prototypes/Recipes/Lathes/security.yml:433]: duplicate

**Review by ficcialfaint — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Recipes/Lathes/security.yml (hunk 1 above)
Comment (Klaypexx) [reply Resources/Prototypes/Recipes/Lathes/security.yml:433]: There is no duplicate, they are differentiated by name ClothingEyesHudSecurity (427) and ClothingEyesHudMedSec (436)

Comment (Klaypexx) [reply Resources/Prototypes/Recipes/Lathes/security.yml:433]: or did I misunderstand something

**Comment by JoeHammad1844** [comment]
just give secglasses sechud like in ss13
Trollface50x50

**Comment by deltanedas** [comment]
knockoff :trollface: :trollface:

**Comment by JoeHammad1844** [comment]
I am god.

**Comment by Klaypexx** [comment]
offtop in dis plz

**Comment by JoeHammad1844** [comment]
#18907 was merged

**Pull request closed without merging**